chingu-voyage4/Geckos-Team-32
# React router, login/signup, structure refactor, styling

## Purpose
* Create login/signup pages

## Approach/Fixes
* Setup react router
* Re-factored file structure to cater to client side page rendering using react router
* Create simple forms for both signup and login page
* Basic styling for new pages
* Made adjustments to the navbar to work with react router
* Made a basic `NotFound` page to handle client side 404 errors

## Notes
* Forms don't have any actions yet (clicking `log in` or `submit` will result in an error
* Page rending is for *CLIENT* side only. Server side rendering is not setup yet (if you try typing in a route on localhost:8080 the GET request will fail)

## Testing
* Download this code to your local device
* NPM install the new react router dependency
* Check that the anchor tags in the navbar work (some will show the NotFound page; the signup and login pages should load properly)
* Check that there is a form and submit button on both the login and signup pages
* Check that clicking the '5GECHO' logo sends you to the `landing page` which @Raymond-Cox and I will be setting up next
